pa11ycrawler/templates/base.html

```diff
@@ -0,0 +1,31 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="assets/css/bootstrap-theme.min.css" />
+        <link rel="stylesheet" href="assets/css/bootstrap-table.min.css">
+        <style>
+            /*An override to prevent the tables from adding scroll bars*/
```

Comment:
Not inaccessible, but I was finding it made the table more difficult to navigate.  The native browser scrollbar works fine unless we care about viewing these tables on mobile devices.  I'm open to arguments that they should be there.